_posts/2015-11-24-hoodie-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Hoodie 1.0 milestones closing in"
+author: Jan
+layout: blog-post-tgif
+comments: true
+---
+
+This is Part 2 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
```

Comment:
I’d say "in the past five months" and leave the "three" out entirely. 

But I don't mind, so if you like it better that way, please leave it in :)